.golangci.yml

```diff
@@ -0,0 +1,17 @@
+run:
+  concurrency: 2
+  timeout: 5m
+  skip-dirs:
+    - pkg/cloud/digitalocean/actuators/machine/machinesetup
+    - pkg/apis
+
+linters:
+  disable-all: true
+  enable:
+    - vet
+    - golint
+    - varcheck
+    - structcheck
+    - gosimple
+    - unused
+    - errcheck
```

Comment:
Looking at other CAPI projects, they enable a bit more linters. Is it possible to enable those here too?